src/entities/identity/model/selectors.ts

```diff
@@ -1,2 +1,14 @@
-export const accessTokenSelector = (state: RootState) =>
-  state.identity.accessToken;
+import { createSelector } from '@reduxjs/toolkit';
+
+import { TokenStorage } from '@shared/lib';
+
+const selectIdentity = (state: RootState) => state.identity;
+
+export const selectUser = createSelector(
+  selectIdentity,
+  identity => identity.user,
```

Comment:
I'd prefer to avoid unnecessary linking on selector level.
Can we use ... => state.identity.user instead? Code would be more obvious and understandable